_studio/shared/umc/codec/vc1_dec/src/umc_vc1_dec_frame_descr.cpp

```diff
@@ -66,11 +66,11 @@ bool VC1FrameDescriptor::Init(uint32_t         DescriporID,
         m_pContext->bp_round_count = -1;
         ptr = (uint8_t*)m_pContext;
 
-        ptr += mfx::align_value<uint32_t>(sizeof(VC1Context));
+        ptr += mfx::align2_value<uint32_t>(sizeof(VC1Context));
         m_pContext->m_picLayerHeader = (VC1PictureLayerHeader*)ptr;
         m_pContext->m_InitPicLayer = m_pContext->m_picLayerHeader;
 
-        ptr += mfx::align_value<uint32_t>((sizeof(VC1PictureLayerHeader)*VC1_MAX_SLICE_NUM));
+        ptr += mfx::align2_value<uint32_t>((sizeof(VC1PictureLayerHeader)*VC1_MAX_SLICE_NUM));
```

Comment:
Do we still need explicit `uint32_t` here?

Reply:
no